rules.ini

```
@@ -194,6 +194,8 @@ FMOD = (?:^|/)(?:lib)?fmod(?:l|ex|exl|studio|studiol)?(?:64)?\.(?:dylib|dll|so)$
 FreeImage = (?:^|/)FreeImage(?:\.so|\.dll)$
 Granny_3D = (?:^|/)granny2(?:_x64)?\.dll$
 Greenworks = (?:^|/)greenworks-(?:win|osx|linux)(?:32|64)?\.node$
+iCue[] = (?:^|/)CgSDK(?:\.x64_2019|\.x64_2017|_2017|\.x64_2015|\.x64_2013|_2019|_2015|_2013)?\.dll$
+iCue[] = (?:^|/)CUESDK(?:\.x64_2017|_2017|\.x64_2015|_2015|_2013)?\.dll$
```

```suggestion
iCue[] = cgsdk
iCue[] = cuesdk
```
won't bring any false positive